packages/reports/src/Components/Form/CompositeCalendar/side-list.tsx

```diff
@@ -1,16 +1,28 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { toMoment } from '@deriv/shared';
 import ListItem from './list-item.jsx';
 
-const isActive = (from, to, flag) => {
+type TItem = {
+    value: string;
+    label: string;
+    onClick: () => void;
+    duration: number;
+};
+
+type TSideList = {
+    from: string;
```

Comment:
@aizad-deriv Why did you change this from the original data type (`number`) to `string`?
```suggestion
    from: number;
```

Reply:
I will change it 🙇🏼  back to the original

Reply:
![prop-type](https://user-images.githubusercontent.com/104334373/210930856-a45a5889-3a82-4c34-9ad3-b20b19e160f3.png)